main.py

```diff
@@ -34,7 +34,7 @@
 def handle_arguments(cl_arguments):
     parser = argparse.ArgumentParser(description='')
     # Configuration files
-    parser.add_argument('--config_file', '-c', type=str, nargs="+",
+    parser.add_argument('--config_file', '-c', type=str, nargs="+", default="config/defaults.conf",
```

Comment:
If setting this default, might want to enforce that `main.py` is run from the main jiant/ directory as well?

Reply:
That sounds pretty non-standard, no? Wouldn't impact many use cases, since you'll nearly always want to use a custom conf.